subiquitycore/models/network.py

```diff
@@ -523,6 +523,13 @@ def render_config(self):
 
         return config
 
+    def rendered_config_paths(self):
+        """Return a list of file paths rendered by this model."""
+        return [
+            '/' + write_file['path']
+            for write_file in self.render().get('write_files').values()
```

Comment:
This also feels a bit repetitive. Maybe we can somehow indicate in the write_files stanza which files should be removed on clean? (Not sure how this would work though)